scripts/docker_release.sh

```diff
@@ -0,0 +1,29 @@
+#/bin/bash
+set -e
+## Sets Wallaroo Docker repo host to Bintray repo.
+wallaroo_docker_repo_host=wallaroo-labs-docker-wallaroolabs.bintray.io
+## Sets repo for First Install Wallaroo Docker image
+wallaroo_docker_image_repo=$wallaroo_docker_repo_host/first-install
+## Sets version variable to first version found in CHANGELOG
+version=`grep -Po '(?<=##\s\[).*(?=\])' CHANGELOG.md | head -1`
+## If version is unreleased, use commit as tag. Otherwise use version.
+if [ $version = "unreleased" ]
+then
+	docker_image_tag=$(git describe --tags --always)
+else
+	docker_image_tag=$version
+fi
+wallaroo_docker_image=wallaroo:$docker_image_tag
+wallaroo_docker_image_path=$wallaroo_docker_image_repo/$wallaroo_docker_image
+## Conditional check for whether current image tag exists in repo, does not
+## re-upload image if so. Otherwise builds docker image and uploads to Bintray.
+if [[ $(docker pull $wallaroo_docker_image_path) == *"Status:"* ]]
```

Comment:
Doing a `pull` in order to check if the tag already exists is very expensive. I'd suggest changing this to rely on the `tag list` api endpoint instead. Doing a get on `https://wallaroo-labs-docker-wallaroolabs.bintray.io/v2/first-install/wallaroo/tags/list` will result in a json response listing all tags which can then be search/filtered using `jq` or something else to check for the existence of the tag.

Reply:
I'll update, thanks for the suggestion